website/www/site/content/en/documentation/runners/basics.md

```diff
@@ -0,0 +1,193 @@
+---
+title: "Basics of the Beam model"
+---
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Basics of the Beam model
```

Comment:
Please rename this file to just `documentation/basics.md`. It's not actually specific to runners now, it's being used as a general introduction the the Beam model.

Reply:
done